app/api/groups/get_breadcrumbs.feature

```diff
@@ -0,0 +1,200 @@
+      | 26              | 9              | 9999-12-31 23:59:59 |
+      | 26              | 10             | 9999-12-31 23:59:59 |
+      | 26              | 11             | 9999-12-31 23:59:59 |
+      | 26              | 13             | 9999-12-31 23:59:59 |
+      | 26              | 14             | 9999-12-31 23:59:59 |
+      | 26              | 16             | 9999-12-31 23:59:59 |
+      | 26              | 19             | 9999-12-31 23:59:59 |
+      | 26              | 22             | 9999-12-31 23:59:59 |
+      | 26              | 25             | 2010-01-01 00:00:00 |
+      | 26              | 27             | 9999-12-31 23:59:59 |
+      | 5               | 41             | 2010-01-01 00:00:00 |
+      | 7               | 41             | 2010-01-01 00:00:00 |
+      | 9               | 41             | 2010-01-01 00:00:00 |
+      | 10              | 41             | 2010-01-01 00:00:00 |
+      | 12              | 41             | 2010-01-01 00:00:00 |
+      | 13              | 41             | 2010-01-01 00:00:00 |
+      | 14              | 41             | 2010-01-01 00:00:00 |
+      | 15              | 41             | 2010-01-01 00:00:00 |
+      | 16              | 41             | 2010-01-01 00:00:00 |
+      | 18              | 41             | 2010-01-01 00:00:00 |
+      | 19              | 41             | 2010-01-01 00:00:00 |
+      | 21              | 41             | 2010-01-01 00:00:00 |
+      | 22              | 41             | 2010-01-01 00:00:00 |
+      | 23              | 41             | 2010-01-01 00:00:00 |
+      | 25              | 41             | 2010-01-01 00:00:00 |
+    And the groups ancestors are computed
+    And the database has the following table 'groups_ancestors':
+      | ancestor_group_id | child_group_id | expires_at          |
+      | 5                 | 41             | 2010-01-01 00:00:00 |
+      | 7                 | 41             | 2010-01-01 00:00:00 |
+      | 9                 | 41             | 2010-01-01 00:00:00 |
+      | 10                | 41             | 2010-01-01 00:00:00 |
+      | 12                | 41             | 2010-01-01 00:00:00 |
+      | 13                | 41             | 2010-01-01 00:00:00 |
+      | 14                | 41             | 2010-01-01 00:00:00 |
+      | 15                | 41             | 2010-01-01 00:00:00 |
+      | 19                | 41             | 2010-01-01 00:00:00 |
+      | 22                | 41             | 2010-01-01 00:00:00 |
+      | 23                | 41             | 2010-01-01 00:00:00 |
+
+  Scenario: Get breadcrumbs
+    Given I am the user with id "41"
+    When I send a GET request to "/groups/9/11/7/16/4/19/27/22/13/14/breadcrumbs"
+    Then the response code should be 200
+    And the response body should be, in JSON:
+    """
+    [
+      {"id": "9", "name": "Managed Class", "type": "Class"},
+      {"id": "11", "name": "Ancestor Team", "type": "Team"},
```

Comment:
I don't know if that's important but this cannot happen as team cannot have a non-user as a child.

Reply:
This team has only one child - the user with id=41. So here we implicitly test that the service doesn't check the hierarchy.